packages/microsoft-reactnative-sampleapps/windows/SampleLibraryCPP/SampleModuleCPP.h

```diff
@@ -16,9 +16,9 @@ namespace SampleLibraryCpp {
 
 // Sample REACT_MODULE
 
-REACT_MODULE(SampleModuleCpp);
-struct SampleModuleCpp {
-  const std::string Name = "SampleModuleCpp";
+REACT_MODULE(SampleModuleCppImpl /*moduleClass*/, L"SampleModuleCpp" /*moduleName*/);
```

Comment:
>/*moduleClass*/ [](start = 33, length = 15)

Seeing the comments inside of macro makes me think what it is going to break. It is better to avoid doing it.